### ISSUE TITLE
Authentication dialog pops up repeatedly, even when already authenticated

### ISSUE THREAD

**Issue by treuss** [LubosD/twinkle#100]
Hi,

when I start Twinkle (v1.10.1 - 07 on Ubuntu 17.04), it would repeatedly ask for the credentials saved in my user profile. If I cancel the autentication dialog, I'm usually already authenticated.

Best regards
Thomas

**Comment by fbriere** [comment]
By "repeatedly", do you mean "once every time Twinkle is started", or does it actually popup more than once during the same run?  And by "usually", do you mean that there are instances where authentication was not performed?

Do you have more than one user profile enabled at startup?  I imagine that having two user profiles, one with authentication info and one without, could trigger such a behavior.  (Twinkle will display the "registered" icon as long as at least _one_ account is working.)

**Comment by treuss** [comment]
Hi @fbriere ,

thanks for answering!
- By _repeatedly_ I meant, multiple times per same run. 
- I have exactly one user profile.

In the meantime I found out what actually caused the problem:
In my company, we're currently migrating from a legacy telephony appliance to a new SIP-based one. I'm one of the users who's already migrated. That's why I'm using Twinkle.
When I first started setting up Twinkle, I put my main contacts in the buddy list, so I could quickly call them. I also put some people in the list who still used the old telephony appliance, What I didn't realize was, that the old devices don't publish any Online Status and that's where the error messages originated!
As soon as I unchecked "Check Online Status", the repeated authentication dialogs disappeared for good.

Thanks again and best regards
Thomas
